### PR TITLE
 Add a orbit bundle for geronimo-kernel version 3.0.1 

### PR DESCRIPTION
**Pull request by GDRDABARERA**
Add a geronimo-kernel version for 3.0.1

### CODE REVIEW

**Comment by madurangasiriwardena** [comment]
@GDRDABARERA shall we change the PR title with a more meaningful one?

File: geronimo-kernel/3.0.1.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+~
+~ Unless required by applicable law or agreed to in writing,
+~ software distributed under the License is distributed on an
+~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+~ KIND, either express or implied.  See the License for the
+~ specific language governing permissions and limitations
+~ under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
```
Comment (DMHP) [inline geronimo-kernel/3.0.1.wso2v1/pom.xml]: Remove unnecessary new line

**Comment by madurangasiriwardena** [comment]
Shall we change the name like "Add orbit bundle for geronimo-kernel version 3.0.1" ?

File: geronimo-kernel/3.0.1.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                        <Bundle-SymbolicName>${pom.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${pom.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.apache.geronimo.kernel.*;version="3.0.1",
+                            org.apache.geronimo.gbean.*;version="3.0.1",
+                        </Export-Package>
+                        <Import-Package>
+                            !org.apache.geronimo.kernel.*,
+                            !org.apache.geronimo.gbean.*,
+                            *;resolution:=optional
```
[the remark this answers is not in the document]
Comment (GDRDABARERA) [reply geronimo-kernel/3.0.1.wso2v1/pom.xml]: At run time bundle will wait till these packages got imported, Hence put resolution:=optional
reference:file:../dropins/geronimo-kernel_3.0.1.wso2v1.jar [53]
  Direct constraints which are unresolved:
    Missing imported package com.thoughtworks.xstream_1.3.0.
    Missing imported package com.thoughtworks.xstream.alias_1.3.0.
    Missing imported package com.thoughtworks.xstream.converters_1.3.0.
    Missing imported package com.thoughtworks.xstream.converters.basic_1.3.0.
    Missing imported package com.thoughtworks.xstream.converters.reflection_1.3.0.
    Missing imported package com.thoughtworks.xstream.core_1.3.0.
    Missing imported package com.thoughtworks.xstream.io_1.3.0.
    Missing imported package com.thoughtworks.xstream.io.xml_1.3.0.
    Missing imported package com.thoughtworks.xstream.mapper_1.3.0.
    Missing imported package net.sf.cglib.asm_2.1.0.
    Missing imported package org.apache.geronimo.crypto_0.0.0.
    Missing imported package org.apache.xbean.osgi.bundle.util_3.11.0.
    Missing imported package org.apache.xbean.osgi.bundle.util.equinox_3.11.0.
    Missing imported package org.apache.xbean.recipe_3.11.0.

**Pull request merged**